concordium-consensus/src/Concordium/KonsensusV1/Types.hs

```diff
@@ -0,0 +1,804 @@
+{-# LANGUAGE BinaryLiterals #-}
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE NamedFieldPuns #-}
```

Comment:
I'm sorry, but can we please not add another record extension. 

It would be nice to use this in general, but the rest of the codebase does not use it. So let's not introduce it in one of 200 modules.

Reply:
Removed NamedFieldPuns.